### Hi there! ðŸ‘‹

<p>I'm studying Cellular and molecular biology at UniversitÃ© de Sherbrooke, in Canada. I have a bachelor's degree in Economics and I speak French, English and German. I am also a craft beer specialist and still have a foot in that sector in my hometown. My rÃ©sumÃ© can be found here: https://bit.ly/resumeJRL. </p>

<p>I'm currently doing an internship in Laforest-Lapointe Lab, where I'm learning to use different tools to process metagenomic (currently shotgun) data from samples to analysis. This lab deals in microbial ecology, so I apply these tools in the context of microbiome studies in humans and plants. I have been testing all kinds of tools and implementing them in pipelines suited to the needs of my colleagues' research and plan on making them available to the public. I also have been leading a research project in microbial ecology which is in its early stages, and I am getting prepared for graduate studies and research.</p>

<p>Since there are so many available tools worthy of a place in a single step of a good metagenomic pipeline, I am also reviewing the literature to understand, and eventually advise on, which tool to use for any given step. Metagenomic research publications often lack a rationale behind the choice of tools, which suggests such a choice is often being made based on what others have done. In such a case, any bias introduced by a given tool spreads across the literature and a tool with a high citation rate may give a false impression of being gold-standard.</p>

<p>I'm currently testing these tools on deeply sequenced shotgun saliva and fecal samples. I have tested and integrated several upstream processing tools that aim at solving similar problems, e.g. reads quality control, taxonomic classification metabolic profiling, and metagenome assembly, binning and clustering. For downstream analysis of such data as will come out of these pipelines, I have also been testing a variety of approaches, widespread and _old school_ ones regarded as standards of ecology, as much as novel and cutting-edge ones adapted to the systemic properties of NGS data. My aim is to help choose among these tools and understand where they disagree and why.</p>

<p>I am particularly interested in metagenome assembly as a tool to tackle the socalled _dark matter_ of metagenomes. From binning and clustering strategies to strain richness exploration, I think it's a fascinating way of looking at community characteristics while always stepping as far away as possible from the biases inherent to reference-based methods, even though in the end, all approaches eventually need references to attribute taxonomy or functional potential.</p>

<p>I am also a strong, albeit amateur, proponent of proper language in microbiome researche. In metagenomics, we are looking not at the abundance of taxa, but that of the reads we attribute them. We are looking not at the diversity of the _gut microbiota_, but that of a _sample's microbiome_ which is posited to be an _estimator_ of it. Although my formal statistical training is not advanced, I understand that properties of NGS data impose limitations on how it can be analysed, a notion that is often not mentioned in so many studies, let alone addressed. I always look forward to discuss these matters with knowledgable people, all the more since there exist different schools of thought on the matter. I hope one day to myself creatively tackle these issues.</p>
  
### A case for deeper benchmarking of bioinformatic tools accuracy
<p>As of 2022, many bioinformatic tools have been reviewed, tested and benchmarked; however, the focus is often on computational requirements. I want to find out which tools are the most accurate and what biases are being introduced by each approach. Although some benchmarking have produced such information for certain tools (especially assemblers and aligners), the answer is often non-trivial, especially with assembly-based (de novo) approaches.</p>

<p>Though it is possible that, one day, gold-standards be recognized in metagenomic bioinformatics, many parameters and tool choices seem to depend on data quality and origin, research objectives, and processing approach choices. Since the field of metagenomics is undergoing an extremely fast evolution, I expect that the best way to help researchers prepare their data for downstream analyses is not only to make the tools accessible, but to guide them in choosing the right tools and parameters thereof.</p> <p>In this regard, my goal is to summarize </p>
<ul>
  <li>which tools are better suited to various data processing task;</li>
  <li>their function and crucial parameters; </li>
  <li>the reviews supporting their accuracy; </li>
  <li>articles supporting the choice of parameters; </li>
  <li>any recommended best practice for their use.</li>
</ul>

<p>I believe that such information will be valuable to any researcher who does not want to blindly rely on preconceived pipelines that may or may not suit their data and scientific objectives. I think any bioinformatic analysis should be understood and thought-through before being run, and therefore believe that a tool-intensive process such as metagenome sequencing data processing should be pondered and understood.</p>  
  
<!--
**jorondo1/jorondo1** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- ðŸ”­ Iâ€™m currently working on ...
- ðŸŒ± Iâ€™m currently learning ...
- ðŸ‘¯ Iâ€™m looking to collaborate on ...
- ðŸ¤” Iâ€™m looking for help with ...
- ðŸ’¬ Ask me about ...
- ðŸ“« How to reach me: ...
- ðŸ˜„ Pronouns: ...
- âš¡ Fun fact: ...
-->
